roles/ansible-test/tasks/init_test_options.yaml

```diff
@@ -40,6 +40,10 @@
         ansible_test_options: "{{ ansible_test_options }} --skip-test {{ ansible_test_sanity_skiptests|join(' --skip-test ') }} "
       when: ansible_test_sanity_skiptests|length > 0
 
+    - name: Reduce verbosity
+      set_fact:
+        ansible_test_verbosity: "-v"
```

Comment:
I don't understand why you unconditionally change the value.

Reply:
It's not unconditional, it's inside a `block:` with `when: ansible_test_test_command == 'sanity'`

Reply:
Oh sorry.